src/style.css

```diff
@@ -1 +1,94 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
```

Comment:
Similar to line 2, the `font-family` property should only specify the custom font name `Avenir-Heavy` without fallback fonts. Fallbacks should be used in other CSS selectors.